.github/workflows/ik-ci.yml

```diff
@@ -0,0 +1,45 @@
+name: InstrumentKit CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install tox tox-gh-actions coverage coveralls
+    - name: Test with tox
+      run: tox
+#    - name: Submit to coveralls
+#      env:
+#        COVERALLS_PARALLEL: true
+#        COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+#        SERIVCE_JOB_ID: ${{ matrix.test_number }}
+#      run: coveralls
+
+    - name: Coveralls
+      uses: AndreMiras/coveralls-python-action@develop
```

Comment:
Neat find

Reply:
Yeah, I was getting a lot of issues with the official coveralls action integration here due to the structure of the report that `pytest --cov=` generates. Apparently the solution is to use the `coverage` command directly, but I want things to be symmetric with tox. The author here solved this problem.

Reply:
Definitely a neat solution